src/main/java/uk/gov/hmcts/reform/sendletter/services/LetterService.java

```diff
@@ -36,7 +38,11 @@ public LetterService(Supplier<IQueueClient> queueClientSupplier,
         this.messageTimeToLive = messageTimeToLive;
     }
 
-    public String send(Letter letter) throws ServiceBusException, InterruptedException, JsonProcessingException {
+    public String send(Letter letter, String serviceName)
+        throws ServiceBusException, InterruptedException, JsonProcessingException {
+
+        Asserts.notEmpty(serviceName, "serviceName");
```

Comment:
Should we say something more about assertion failure?

Reply:
it expects parameter name:
```
if (TextUtils.isEmpty(s)) {
            throw new IllegalStateException(name + " is empty");
        }
```